src/file/file.rpc.ts

```diff
@@ -75,111 +97,73 @@ export class FileServer {
     const ownerID: string = call.request.ownerID;
     const parent: string = call.request.parent;
     const size: number = parseInt(call.request.size, 10);
-    FileService.createUpload(
-      key,
-      bucket,
-      name,
-      ownerID,
-      parent,
-      size)
-      .then((upload) => {
-        callback(null, upload);
-      }).catch(err => callback(err));
+    return await FileService.createUpload(key, bucket, name, ownerID, parent, size);
   }
 
   // Updates the uploadID.
   private async updateUpload(call: any, callback: any) {
     const key: string = call.request.key;
     const uploadID: string = call.request.uploadID;
     const bucket: string = call.request.bucket;
-    FileService.updateUpload(
-      uploadID,
-      key,
-      bucket)
-      .then((upload) => {
-        callback(null, upload);
-      }).catch(err => callback(err));
+    return await FileService.updateUpload(uploadID, key, bucket);
   }
 
   // Get an upload metadata by its id in the DB.
   private async getUploadByID(call: any, callback: any) {
     const id = call.request.uploadID;
-    FileService.getUploadById(id)
-      .then((upload) => {
-        callback(null, upload);
-      }).catch(err => callback(err));
+    return await FileService.getUploadById(id);
   }
 
   //  Delete an upload from the DB by its id.
   private async deleteUploadByID(call: any, callback: any) {
     const id = call.request.uploadID;
-    FileService.deleteUpload(id)
-      .then((upload) => {
-        callback(null, upload);
-      }).catch(err => callback(err));
+    return await FileService.deleteUpload(id);
   }
 
   // ********************* FILE FUNCTIONS ********************* */
 
   // Creates a new file in the DB.
   private async createFile(call: any, callback: any) {
     const params = call.request;
-
-    FileService.create(
-      params.bucket,
-      params.name,
-      params.ownerID,
-      params.type,
-      params.parent,
-      params.key,
-      parseInt(params.size, 10))
-      .then((file) => {
-        callback(null, new ResFile(file));
-      })
-      .catch(err => callback(err));
+    return new ResFile(await FileService.create(
```

Comment:
The indentation is incorrect and you shouldn't pass an `await`ed value straight into a function's call, variables aren't expensive...
It should be something like:
```
const createdFile = await FileService.create(
	params.bucket,
	params.name,
	params.ownerID,
	params.type,
	params.parent,
	params.key,
	parseInt(params.size, 10),
);
		
return new ResFile(createdFile);
```